### PR TITLE
Components

### PR DESCRIPTION
**Pull request by khnguyen94**

### CODE REVIEW

**Comment by khnguyen94** [comment]
Added a scrolling bar to sidebar

File: client/src/components/SideBar/SideBar.js [hunk 1]
```diff
@@ -2,88 +2,117 @@ import React from "react";
...
+  };
 
-  // Function to render all the favorite teams from faveTeamsFiltered
-  let renderFaveTeams = props.teams
-    .filter(team => {
-      return team.favorite === true;
+  // Function to handle when a favorite button for an already favorited team is clicked
+  // faveTeams ==> allTeams
+  handleUnfavoriteSubmit = (event, id, checked) => {
+    event.preventDefault();
```
Comment (metacmm) [inline client/src/components/SideBar/SideBar.js]: should call API.removeTeamFromFavorites(id)

File: client/src/components/SideBar/SideBar.js [hunk 2]
```diff
@@ -2,88 +2,117 @@ import React from "react";
...
+    API.updateTeamFavorite({
```
Comment (metacmm) [inline client/src/components/SideBar/SideBar.js]: should call API.removeTeamFromFavorites(id)

File: client/src/components/SideBar/SideBar.js [hunk 3]
```diff
@@ -2,88 +2,117 @@ import React from "react";
...
+            </a>
+            <FavoriteTeamBtn onClick={() => this.handleNewFavoriteSubmit(team.id)}/>
+          </li>
+        );
+      });
+  };
+
+  // Function to render all the favorite teams from faveTeamsFiltered
+  renderFaveTeams = () => {
+    props.teams
```
Comment (metacmm) [inline client/src/components/SideBar/SideBar.js]:
```
API.getFavoriteTeam.filter(...)
```

**Pull request merged**